tiddlers/$__StoryList.tid

```diff
@@ -1,6 +0,0 @@
-created: 20181111055700759
```

Comment:
Looks like you've deleted the files while adding the ignores.

Reply:
Yes, I thought this was the quickest way to go.
I'll ignore them and mark them as assume-unchanged.

Reply:
And revert the delete.